symplyphysics/laws/electricity/radius_of_a_charged_particle_in_a_magnetic_field.py

```diff
@@ -28,6 +30,23 @@
 
 law = Eq(radius, mass * velocity / (charge * induction))
 
+# This law might be derived via period of a charged particle in a magnetic field.
+
+#TODO:
+## The following law should be added:
+## T = 2 * pi * radius / velocity
```

Comment:
Well, we have `T = 2 * pi / w` law, so only `w = v / r` missing. But you can simply use `distance_from_constant_velocity` law, given distance is a full circle = `2 * pi * r` and velocity is constant. 
But you should mention the definition of period: it is a time taken to cover a full circle.